### ISSUE TITLE
URGENT - 025 NCSM - Unable to upload file

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#549]
In GitLab by @kdjstudios on Nov 2, 2017, 12:48

**Submitted by:** "Jesus Corchado" <jesus.corchado@answernet.com>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2017-11-02-68696/conversation

**Server:** Internal

**Client/Site:** NCSM

**Account:** All

**Issue:** 

I tried uploading the file to process billing but I get the message below.

> "The Page you are looking for does not exist"

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 2, 2017, 12:50

@tim.traylor @kace.coady 

I have advised Jesus to rerun their export and try uploading again. If that is not successful, I have asked him to send in their upload file. 

If that is not successful, is this something @tim.traylor can complete or will I need to reach out to the Aavaz Team via Whatsapp?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 2, 2017, 13:33

Jesus was able to try uploading again, however this time he received the "Something went wrong" error. I have requested for him to send in the file they are attempting to upload.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 2, 2017, 14:09

Here is the usage file :

[DFT_11012017.txt](/uploads/dc19d43c7115f39e96058a5da2a517bb/DFT_11012017.txt)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 2, 2017, 16:24

@smasih 

Please look into this first this; the NCSM site needs to be able to run their billing. @tim.traylor was unavailable to aid in this today.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Nov 3, 2017, 02:10

Hi [Kyle] ( @kdjstudios ),   
Firstly, thanks for the usage file. It helps us a lot.  
We dug on the issue & found the bug.  
Previously, we have deleted the VCC activities from the NCSM site but there was an account **Cross Country Home Services** which was having the VCC activity -  **VCC Talk Minutes** on it, however, it was disabled from account although it's misbehaving.  
Now, we have deleted the **VCC Talk Minutes** activity from the account & also uploaded the usage.  
**We have uploaded the usage file on the NCSM ``1st Billing Cycle``**, if they want then they can revert & apply again.  
Thanks.
